### PR TITLE
Add information on new vendor setup

### PR DESCRIPTION
**Pull request by dstrain115**
- Add a blurb about submitting RFC and attending cirq cync
- Adds information about possible options and different pieces
needed for an integration.
- Links to this in two places.

Fixes: #3820

### CODE REVIEW

File: docs/dev/rfc_process.md [hunk 1]
```diff
@@ -105,5 +105,46 @@ While we encourage and celebrate every contributor, the bar for RFC acceptance i
...
+amount of coordination and effort but allows a tighter integration and for
+cirq-maintainers to modify the integration code to stay in sync with an
+evolving cirq-core code base.  We generally do not accept submissions for
+this type of integration for third-party vendors that function as
+intermediaries for other cloud vendors.
+
+Several things are needed for a successful integration:
+
+*    Plan for including external dependencies (if needed).
+*    Access and authentication
```
Comment (dabacon) [inline docs/dev/rfc_process.md:140]: nit missing "." here and one below.

File: docs/dev/rfc_process.md [hunk 2]
```diff
@@ -105,5 +105,46 @@ While we encourage and celebrate every contributor, the bar for RFC acceptance i
...
+If this process is functioning well, RFCs are expected to fail in the earlier, rather than later stages.
 An approved RFC is not a commitment to implementation on any sort of timeline. The prioritization of features depends on user interest and willingness of contributors to implement them.
+
+## New hardware integrations
+
+Several hardware vendors already have integrations with cirq.  We are not
+currently soliciting additional vendors.  However, if you are considering
+integrating with cirq, we would highly encourage you to engage with the
+cirq-maintainer team through attending the weekly cirq cync and submitting
+an RFC as specified above.  Everyone benefits from a well-maintained, user
```
Comment (dabacon) [inline docs/dev/rfc_process.md:117]: do we have an example of a vendor integration that would be useful to link to?

**Review by dabacon — APPROVED** [review]
LGTM, one nit and a question

File: docs/dev/rfc_process.md (hunk 1 above)
Comment (dstrain115) [reply docs/dev/rfc_process.md]: Done.

File: docs/dev/rfc_process.md (hunk 2 above)
Comment (dstrain115) [reply docs/dev/rfc_process.md:117]: Linked to hardware pages.  I was reluctant to pick one in particular.

**Pull request merged**